### DISCUSSION TITLE
General anoucement

### DISCUSSION THREAD

**Discussion by BarySax** [BarySax/v2-Abyssal-Sacrament#19] [Announcements]
Discussion start here

**Comment by NicolasPrigge** [comment]
this his a very good game

**Reply by BarySax** [reply]
Firstly, you made a mistake but I don’t care, we aren’t English so. Also it’s not even started, it’s only the beginning.

**Reply by NicolasPrigge** [reply]
thank you for the feedback

**Comment by NicolasPrigge** [comment]
We need to write the basic of the story and we will be able to creat the first build
